test/test_data.py

```diff
@@ -5,91 +5,388 @@
+    np.testing.assert_allclose(streams[i]["clock_values"], clock_values)
+
+    # Stream ID: 0x02C0FFEE
+    i = 1
+    assert streams[i]["info"]["name"][0] == "SendDataString"
+    assert streams[i]["info"]["type"][0] == "StringMarker"
+    assert streams[i]["info"]["channel_count"][0] == "1"
+    assert streams[i]["info"]["nominal_srate"][0] == "10"
+    assert streams[i]["info"]["channel_format"][0] == "string"
+    assert streams[i]["info"]["created_at"][0] == "50942.723319709003"
+    assert streams[i]["info"]["desc"][0] is None
+    assert streams[i]["info"]["uid"][0] == "xdfwriter_11_str"
 
-        clock_times = np.asarray([])
-        clock_values = np.asarray([])
+    # Info added by pyxdf
+    assert streams[i]["info"]["stream_id"] == 0x02C0FFEE
+    assert streams[i]["info"]["effective_srate"] == pytest.approx(10)
+    assert streams[i]["info"]["segments"] == [(0, 8)]
 
-        np.testing.assert_array_equal(streams[1]["clock_times"], clock_times)
-        np.testing.assert_array_almost_equal(streams[1]["clock_values"], clock_values)
+    # Footer
+    assert streams[i]["footer"]["info"]["first_timestamp"][0] == "5.1"
+    assert streams[i]["footer"]["info"]["last_timestamp"][0] == "5.9"
+    assert streams[i]["footer"]["info"]["sample_count"][0] == "9"
+    first_clock_offset = streams[i]["footer"]["info"]["clock_offsets"][0]["offset"][0]
+    assert first_clock_offset["time"][0] == "50979.76"
+    assert first_clock_offset["value"][0] == "-.01"
 
-        streams, header = load_xdf(
-            file, jitter_break_threshold_seconds=0.001, jitter_break_threshold_samples=1
-        )
+    # Time-series data
+    s = [
+        [
+            '<?xml version="1.0"?><info><writer>LabRecorder xdfwriter'
+            "</writer><first_timestamp>5.1</first_timestamp><last_timestamp>"
+            "5.9</last_timestamp><sample_count>9</sample_count>"
+            "<clock_offsets><offset><time>50979.76</time><value>-.01</value>"
+            "</offset><offset><time>50979.86</time><value>-.02</value>"
+            "</offset></clock_offsets></info>"
+        ],
+        ["Hello"],
+        ["World"],
+        ["from"],
+        ["LSL"],
+        ["Hello"],
+        ["World"],
+        ["from"],
+        ["LSL"],
+    ]
+    t = np.array([5.1, 5.2, 5.3, 5.4, 5.5, 5.6, 5.7, 5.8, 5.9])
+    assert streams[i]["time_series"] == s
+    np.testing.assert_allclose(streams[i]["time_stamps"], t)
+
+    # Clock offsets - does not match the footer but it is in the file
+    clock_times = np.asarray([])
+    clock_values = np.asarray([])
+
+    np.testing.assert_equal(streams[i]["clock_times"], clock_times)
+    np.testing.assert_equal(streams[i]["clock_values"], clock_values)
+
+    streams, header = load_xdf(
+        path,
+        synchronize_clocks=synchronize_clocks,
+        dejitter_timestamps=dejitter_timestamps,
+        jitter_break_threshold_seconds=0.001,
+        jitter_break_threshold_samples=1,
+    )
```

Comment:
Is it necessary to load the file a second time? Can you adapt the test so that we only need to load it once at the very top?

Reply:
That was just how it was implemented in the original test, but yes it probably should be it's own separate test.

Reply:
Or we keep it in this test, because we already have a parameterization of `dejitter_timestamps`. However, we might have to adapt some tests, because this specific part seems to rely on the other two arguments set to these exact values.

If it's too complicated, then let's move it to a new test.

Reply:
I've separated this into a separate test, I think it makes more sense.